sdk/communication/azure-communication-rooms/swagger/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure Communication Service room client library for Java
+
+> see https://aka.ms/autorest
+## Getting Started
+
+To build the SDK for Rooms Client, simply Install AutoRest and in this folder, run:
+
+### Setup
+```ps
+Fork and clone https://github.com/Azure/autorest.java
+git checkout main
+git submodule update --init --recursive
+mvn package -Dlocal
+npm install
+npm install -g autorest
+```
```

Comment:
We don't need this section. We don't need to clone the AutoRest source and build locally to generate code.

Reply:
Addressed.